R/pkg/R/sparkR.R

```diff
@@ -344,6 +344,7 @@ sparkRHive.init <- function(jsc = NULL) {
 #' @note sparkR.session since 2.0.0
 sparkR.session <- function(
   master = "",
+  deployMode = "",
```

Comment:
We should move this to the end of the param list as @felixcheung says. On a different note, is there a reason this should be a parameter to `spark.session` ? It would be good if the users didn't have to think of deploy mode and we inferred it from the spark-submit options.